### DISCUSSION TITLE
How to prevent automatic file save dialog?

### DISCUSSION THREAD

**Discussion by catronomix** [EnviralDesign/UberGui#43] [Q&A]
Hi, I very much enjoyed using ubergui for the first time, but when it's in use, every minute it's asking me if I want to overwrite the file 'container_UG_V4.tox. that it tries to auto-save to the desktop. I can't find an option to disable it and it's very distracting

**Comment by EnviralDesign** [comment]
Hi thanks for the issue, what os and td version are you using? Do you have any tool in your toe that attempts to auto save externalized tox files? 

Regardless of what is causing that prompt, you should be able to just clear out the external tox path from the UG comp, and I imagine it will go away:

![image](https://user-images.githubusercontent.com/10091486/118709358-79452a80-b7e2-11eb-8222-c44720ddfaca.png)
